examples/custom-node/src/primitives/extended_op_tx_envelope.rs.rs

```diff
@@ -0,0 +1,276 @@
+use op_alloy_consensus::OpTxEnvelope;
+use alloy_rlp::{BufMut, Decodable, Encodable, Result as RlpResult};
+use alloy_consensus::Transaction;
+use alloy_primitives::{ChainId, TxHash, PrimitiveSignature};
+use reth_primitives_traits::{
+    serde_bincode_compat::SerdeBincodeCompat, transaction::signed::RecoveryError, InMemorySize,
+    SignedTransaction,
+};
+use reth_codecs::Compact;
+use reth_optimism_primitives::{
+    serde_bincode_compat::OpTransactionSigned as BincodeCompatOpTransactionSigned,
+    OpTransactionSigned, OpTxEnvelope,
+};
+use revm_primitives::{Address, Bytes, TxKind, U256};
+use alloy_eips::{eip2930::AccessList, eip7702::SignedAuthorization};
+use revm_primitives::B256;
+
+pub enum ExtendedOpTxEnvelope<T> {
+    BuiltIn(OpTxEnvelope),
+    Other(OpTransactionSigned),
+}
+
+#[derive(Debug, Clone, serde::Serialize, serde::Deserialize, Hash, Eq, PartialEq)]
```

Comment:
I think this has been moved accidentally?

Reply:
yup, moved to correct place